contrib/forms/leg_length/templates/general_new.html

```diff
@@ -2,7 +2,7 @@
 <head>
 {literal}
 
- <style type="text/css" title="mystyles" media="all">
+ <style title="mystyles" media="all">
```

Comment:
This will probably later on need to be revised -- title attributes here are unnecessary and the media attribute is dead in favor of doing it inside the stylesheet